tests/utils_test.py

```diff
@@ -0,0 +1,39 @@
+"""Unit tests for english language checker"""
+from libcipher import utils as utils
+import io
+import sys
+import unittest
+
+
+def get_marys_dictionary():
+        english_words = ["had", "a", "little", "lamb"]
+        if sys.version < '3':
+            dict_file = io.StringIO(unicode('\n').join(english_words))
+        else:
+            dict_file = io.StringIO("\n".join(english_words))
+        return dict_file
```

Comment:
OK, now I understand the problem, it has got to do with `io.StringIO` class. Will let you know how this can be fixed.